### DISCUSSION TITLE
How can I change the pagination links (next and previous page) from http to https?

### DISCUSSION THREAD

**Discussion by yusef22** [spatie/laravel-json-api-paginate#53] [Q&A]
In the configuration file(json-api-paginate.php) I have set base_url = env ('APP_URL'). But the links for next & previous page have a problem. For example, my API is as follows:
https://example.com/api/categories?filter[name]=internet&page[size]=10
But the address of the next page is as follows:
https://example.com/?page[size]=10&page[number]=2
please help me.

**Comment by localpath** [comment]
I have the same issue. In development/local it works as expected, and our links have https, but once in production our links are http.

I have used this in `AppServiceProvider` to force https link generation, maybe it will work for you.
```php
   /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        URL::forceScheme('https');
    }
```

The .env `APP_URL=https://mydomain.com`
